### ISSUE TITLE
✅ 🚀 Feature Request: Spoiler Button (Reveal/Unfold)

### ISSUE THREAD

**Issue by paaljoachim** [elementor/elementor#880]
It would be nice to extend the options in how the accordion is opened.
For instance put some more visual hint options into Style.

Upload your own image to be clicked to open the accordion item.
Turn off the visual hint and have the user click the title used to open the accordion.
There are probably other options as well that would be nice to add.

I am thinking that it would be good to go through each element and see where it is logical to extend it further.

**Comment by arielk** [comment]
I think this is an interesting feature, but I would build for this a unique widget instead of the extend the original accordion widget. For example we can call it a "spoiler button" or something like this..

**Comment by paaljoachim** [comment]
Yeah that is possible. Just creating a new "spoiler button". 
Click the text/image to open a message. Add as many as you like similar to the accordion.
It seems almost like a criss cross between a FAQ, tabs and accordion widgets.
It would be great to brain storm it here when you guys begin internally discussing it. I most likely have other ideas for it as well that will come up along the way.

**Comment by paaljoachim** [comment]
Btw It would also be good to have the spoiler/accordion open sideways and not just vertically.
For instance having a list

Option 1
Option 2
Option 3
Option 4

- Add an icon before the options or after.
- Click to have options open downward or sideways.

**Comment by gahawkey** [comment]
Yes @paaljoachim especially your example of "Click to have options open downward or sideways."

This could be really nice to have when you have a section with a background image, and the list on the left can open panels to the right over the background image... like big tooltips 😃

**Comment by Chinoman10** [comment]
Hello.
2 years later... did this feature request get forgotten?
Thanks in advance.

**Comment by 79ho3ein** [comment]
+1

**Comment by Trystan-SA** [comment]
Need this. It would be awesome if we could insert any elements inside the spoiler (so not just only the text editor).

**Comment by nicholaszein** [comment]
Hey!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with the introduction of the Nested Accordion in Elementor v3.15.0! 🥳 

Learn more about it here: https://elementor.com/help/accordion-widget/

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
